python/constants.cpp

```diff
@@ -1,107 +0,0 @@
-#ifdef NANOGUI_PYTHON
```

Comment:
As discussed above, this file is now gone in favor of two (one for GLFW, one for Entypo)